src/services/CoresService.Video.cs

```diff
@@ -58,13 +85,50 @@ public void AddDisplayModes(string identifier)
     {
         var info = this.ReadCoreJson(identifier);
         var video = this.ReadVideoJson(identifier);
-        List<DisplayMode> all = ALL_MODES.Select(id => new DisplayMode { id = id }).ToList();
+        List<DisplayMode> all = new List<DisplayMode>();
+
 
         if (info.metadata.platform_ids.Contains("gb"))
         {
             all.AddRange(GB_MODES.Select(id => new DisplayMode { id = id }));
         }
 
+        if (info.metadata.platform_ids.Contains("gbc"))
+        {
+            all.AddRange(GBC_MODES.Select(id => new DisplayMode { id = id }));
+        }
+
+        if (info.metadata.platform_ids.Contains("gba"))
+        {
+            all.AddRange(GBA_MODES.Select(id => new DisplayMode { id = id }));
+        }
+
+        if (info.metadata.platform_ids.Contains("gg"))
+        {
+            all.AddRange(GG_MODES.Select(id => new DisplayMode { id = id }));
+        }
+
+        if (info.metadata.platform_ids.Contains("lynx"))
+        {
+            all.AddRange(LYNX_MODES.Select(id => new DisplayMode { id = id }));
+        }
+
+        if (info.metadata.platform_ids.Contains("ngp"))

```

Comment:
This is mildy annoying: https://github.com/jotego/jtbin/blob/master/pocket/raw/Cores/jotego.jtngp/core.json#L5-L8